.circleci/config.yml

```diff
@@ -180,12 +184,13 @@ references:
       paths:
         - project
 
-  Secrets: &Secrets
+  secrets: &secrets
```

Comment:
why did you lower case it? i think it is better shown in the step as `Secrets`

Reply:
![image](https://user-images.githubusercontent.com/46294017/87524096-6dec1480-c690-11ea-95ac-c1744440bb44.png)
In the step itself, it is capitalized - I only changed the alias to be lowercase in keeping with the format of all of the other aliases.